doc/Doxyfile.in

```diff
@@ -140,7 +140,7 @@ INLINE_INHERITED_MEMB  = NO
 # shortest path that makes the file name unique will be used
 # The default value is: YES.
```

Comment:
Shouldn't this line be changed as well then to "# The default value is: NO." ?

Reply:
That's referring to the upstream default, no?